sass/px-app-nav.scss

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2014 GE Global Research. All rights reserved.
+ *
+ * The copyright to the computer software herein is the property of
+ * GE Global Research. The software may be used and/or copied only
+ * with the written permission of GE Global Research or in accordance
+ * with the terms and conditions stipulated in the agreement/contract
+ * under which the software has been supplied.
+ */
+
+// Settings
+@import "px-colors-design/_settings.colors.scss";
+
+// Generic
+@import "px-box-sizing-design/_generic.box-sizing.scss";
+
+// Base
+@import "px-typography-design/_base.typography.scss";
+
+// Component
+@function rem($size) {
```

Comment:
curious as to why you set your own rem function that is different than the one that we use in px-functions-design: https://github.com/PredixDev/px-functions-design/blob/master/_tools.functions.scss#L33